### ISSUE TITLE
master: regression 

### ISSUE THREAD

**Issue by kohlhase** [slatex/sTeX#278]
On `.../MathHub/MiKoProblems/AIplenary/source/complexity/en/complexity-classes.tex` I get new errors that are illigitimate:
```
(/Users/kohlhase/localmh/MathHub/MiKoMH/AI/source/complexity/tikz/common-landau-sets.tex

/Users/kohlhase/localmh/MathHub/MiKoMH/AI/source/complexity/tikz/common-landau-sets.tex:7: Package modules Error: `\termref' with unidentified cd "timespace-complexity".

See the modules package documentation for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.7 ...espace-complexity?constant-class]{constant}
                                                      & 1  &

/Users/Kohlhase/localmh/MathHub/MiKoMH/AI/source/complexity/tikz/common-landau-sets.tex:8: Package modules Error: `\termref' with unidentified cd "timespace-complexity".

See the modules package documentation for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
```

**Comment by Jazzpirate** [comment]
fixed in 096d7534fda188e5cebe6bd7ccbc6a4166cc0fbd

**Comment by kohlhase** [comment]
Thanks, I can verify that this solves my problem.
